libraries/SITL/SITL.cpp

```diff
@@ -1029,6 +1038,11 @@ const AP_Param::GroupInfo SIM::var_ins[] = {
     AP_GROUPINFO("GYR5_BIAS",    47, SIM, gyro_bias[4], 0),
 #endif
 
+    // @Param: OH_RELY_MSK
```

Comment:
I keep reading this as the "Oh, really?" mask.  Just me?

Reply:
:rofl: You're not the only one. `RELY` is the name for the `AP_Relay` log entry. Though I think I have one letter spare for the parameter name - I could probably make it `RELAY`.